### PR TITLE
adding a gulpfile

### LINKED ISSUE #3: setup gulpfile

### PR DESCRIPTION
**Pull request by chenshay**
close #3

### CODE REVIEW

File: build-system/tasks/bundle.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
+/**
```
Comment (erwinmombay) [inline build-system/tasks/bundle.js]: this file can be fully removed

File: build-system/tasks/highlight.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+/**
```
Comment (erwinmombay) [inline build-system/tasks/highlight.js]: remove file

File: build-system/tasks/bundle.js (hunk 1 above)
Comment (chenshay) [inline build-system/tasks/bundle.js]: done

File: build-system/tasks/highlight.js (hunk 2 above)
Comment (chenshay) [inline build-system/tasks/highlight.js]: done

**Pull request closed without merging**